src/main/scala/org/virtuslab/akkasaferserializer/CborAkkaSerializer.scala

```diff
@@ -0,0 +1,68 @@
+package org.virtuslab.akkasaferserializer
```

Comment:
The only problem I can see with this name is its acronym :trollface: 

So probably `akka-borer-serializer` will be a better choice for PR/marketing reasons xD

Reply:
(in all seriousness, #11 opened)